react/week1/todo-list-app/src/App.js

```diff
@@ -0,0 +1,38 @@
+import "./App.css";
+import getTodos from "./getTodos";
+
+const todos = getTodos();
+
+// defining the React props variables
+const ToDos = (props) => {
+  return (
+    <ul type="square">
+      <li>
+        <h3>Task: {props.todoDescription}</h3>
+        <p style={{ color: "red" }}>Deadline: {props.deadline}</p>
+      </li>
+    </ul>
+  );
+};
+
+// Accessing each todo of the above  todo array
+export default function App() {
+  return (
+    <div className="App">
+      <h2 style={{ color: "darkblue" }}> Todo List:</h2>
+      <h4 id="header">Todo's and the deadlines :- </h4>
+      <ul id="todoList">
+        {todos.map((todo) => {
+          return (
+            <ToDos
+              todoDescription={todo.todoDescription}
+              deadline={todo.deadline}
+            />
```

Comment:
Just a small comment:
The console is showing a warning that says `Each child in a list should have a unique "key" prop.`

you can avoid the problem by simply providing a uniqe `key` that have the value of `todo.id` (you also have to add a `id` field in your `getTodos`) 

```suggestion
            <ToDos
              key={todo.id}
              todoDescription={todo.todoDescription}
              deadline={todo.deadline}
            />
```

You will learn more about it in the next weeks :)

Reply:
ok Thanks a lot , i will try to explore ore about key, thanks a  lot for the suggestiom